packages/odt2txt/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/dstosberg/odt2txt
+TERMUX_PKG_DESCRIPTION="simple converter from OpenDocument Text to plain text."
+TERMUX_PKG_VERSION=0.5
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=23a889109ca9087a719c638758f14cc3b867a5dcf30a6c90bf6a0985073556dd
+TERMUX_PKG_SRCURL=https://github.com/dstosberg/odt2txt/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_DEPENDS="libandroid-support, libzip"
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_make () {
+    make \
+        CC="$CC" \
+        CFLAGS="$CFLAGS $CPPFLAGS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64" \
+        LDFLAGS="$LDFLAGS"
```

Comment:
This is not substantially different than the existing Make step, you can just pass any extra variables you want with `TERMUX_PKG_EXTRA_MAKE_ARGS` instead of overriding it.

Reply:
Ok again I copy/pasted remove and package works fine cf https://github.com/termux/termux-packages/pull/3433/commits/1a5b7fbaa7517efec2696bc9a16db0daeb4c2b13